src/components/base/containers/Tooltip.js

```diff
@@ -0,0 +1,205 @@
+import React, { Component, PropTypes } from 'react';
+import { findDOMNode } from 'react-dom';
+import Overlay from 'react-overlays/lib/Overlay';
+import Transition from 'react-overlays/lib/Transition';
+import styled, { injectGlobal } from 'styled-components';
+import { colors } from '../../theme';
+
+injectGlobal(`
```

Comment:
Are you sure you want to injectGlobal here? This code will get run any time this module is imported and will result in a bunch of additional ``transition-in`` and ``transition-out`` classes being put into style tags. Do these classes need to have the styles attached?